### DISCUSSION TITLE
Orietação a Objetos

### DISCUSSION THREAD

**Discussion by JonatasSilva01** [balta-io/forum#708] [Tira Dúvidas] [answered]
Olá professor balta tudo bem? como vai o trabalho? aqui vai bem ! se pode corrigir essa dúvida?

Para classes parciais seria inteligente da minha parte caso encontre um arquivo com mais de 100 linhas de códigos separar as propriedades, metodos e eventos em classes separadas por exemplo:

Exemplo.Metodos.cs
Exemplo.Propriedades.cs
Exemplo.Eventos.cs

AULA - Partial Class

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @JonatasSilva01, como vai?

É sábio sim dividir sua classe em classes menores, mas não necessariamente propriedades, métodos e eventos.
Mais adiante (Lá no fim dos cursos) você verá a parte de Value Objects que vai te ajudar com isto.

Mesmo neste caso, Partials não são os mais indicados... Você vai ter uma compreensão maior assim que terminar todas as carreiras, fique tranquilo.

**Reply by JonatasSilva01** [reply]
Olá professor balta, eu to bem sim! estou no trabalho resolvendo alguns pepinos haha... certamente eu vou dominando esses assuntos em seu video e lendo algumas coisas, acredito que quando fizer os primeiros projetos e dominar algumas praticas eu vou me destacando mais facilmente.
